### ISSUE TITLE
Video Transcoding Service

### ISSUE THREAD

**Issue by Mrinank-Bhowmick** [Mrinank-Bhowmick/python-beginner-projects#605]
## 🚀 Feature

**Describe the solution you'd like**
We need a simple video transcoding service that can take raw video input and convert it to different formats like 1080p, 480p, etc.

## 📝 Additional context

**Implementation Details**
- The service should accept raw video files as input.
- The service should be able to convert these videos into various formats such as 1080p, 480p, etc.
- The service should maintain the aspect ratio of the original video during the conversion process.
- The service should provide meaningful error messages for invalid inputs.

This is an open issue for all, so mention this issue in your pull requests for keeping a track.

**Comment by bhandareasmita** [comment]
@Mrinank-Bhowmick, are we starting from scratch with a new project, or do we have an existing codebase or starting point to build upon?"

**Comment by Mrinank-Bhowmick** [comment]
>  @Mrinank-Bhowmick, are we starting from scratch with a new project, or do we have an existing codebase or starting point to build upon?"

New project, start from scratch

**Comment by shashaaankkkkk** [comment]
can i join this @Mrinank-Bhowmick

**Comment by Mrinank-Bhowmick** [comment]
> can i join this @Mrinank-Bhowmick

yeah sure

**Comment by shashaaankkkkk** [comment]
hey i also want to add some flask and frontend to it

**Comment by AyushSenDazzler** [comment]
Hey @Mrinank-Bhowmick! Can you assign me this issue? I'm keen to work on this project. @shashaaankkkkk If you have made some progress, let me know about your approach. I'm well-versed in FLASK and JS.

**Comment by Mrinank-Bhowmick** [comment]
Yeah sure collaborate with each other @AyushSenDazzler @shashaaankkkkk

**Comment by shashaaankkkkk** [comment]
yes sure lets work together @Mrinank-Bhowmick @AyushSenDazzler

**Comment by hudihi** [comment]
@Mrinank-Bhowmick does this project requires to make a simple ui maybe and use python to make this functionality works

**Comment by Mrinank-Bhowmick** [comment]
@hudihi Yes, this project could benefit from a simple UI for better usability
